### PR TITLE
fix: Fix checkout styling issues

### PR DESCRIPTION
**Pull request by soniaklimas**
I want to merge this change because it fixes checkout styling issues.

# Pull Request Checklist

- [ ] Test the changes locally to ensure they work as expected.
- [ ] Document the testing process and results in the pull request description. (Screen recording, screenshot etc)
- [ ] Include new tests for any new functionality or significant changes.
- [ ] Ensure that tests cover edge cases and potential failure points.
- [ ] Document the impact of the changes on the system, including potential risks and benefits.
- [ ] Provide a rollback plan in case the changes introduce critical issues.
- [ ] Update documentation to reflect any changes in functionality.

### CODE REVIEW

**Review by gajda-w — APPROVED** [review]

**Review by GrzegorzDerdak — CHANGES_REQUESTED** [review]
Skeleton also needs to be improved, because during loading it's not stretched. Also, the solution does not look properly.

File: .vscode/settings.json [hunk 1]
```diff
@@ -19,5 +19,8 @@
     "bradlc.vscode-tailwindcss"
   ],
   "cSpell.words": ["nimara", "saleor", "undiscounted"],
-  "editor.defaultFormatter": "esbenp.prettier-vscode"
+  "editor.defaultFormatter": "esbenp.prettier-vscode",
+  "debug.javascript.defaultRuntimeExecutable": {
+    "pwa-node": "/Users/soniaklimas/.local/share/mise/shims/node"
+  }
```
Comment (GrzegorzDerdak) [inline .vscode/settings.json:25]: 🥇

**Review by GrzegorzDerdak — APPROVED** [review]
Looks good to me, but please remove the extra lines from `.vscode/settings.json`
